Woof/Woof/View/ReusableViews/FiveStarRatingView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+/// A view displaying a rating of the person.
+struct FiveStarRatingView: View {
+    // MARK: Private interface
+
+    /// The text to display if the rating is not specified.
+    private let noRatingText = "not rated"
+
+    // MARK: Internal interface
+
+    /// The rating in the range from 1 to 5 stars.
```

Comment:
But according to the code, the rating can have `0` value.
What does that mean if the rating equals to `0`? Should we document it?

Reply:
> Should we document it?

Do you mind answer the question? :)
The doc comment must match the current behaviour.

Reply:
Yes, sure. You are right. According to the code, the rating can have `0` value.
Fixed documentation.